app/src/main/res/values/strings.xml

```diff
@@ -380,6 +382,10 @@
     <string name="nc_share_this_location">Share this location</string>
     <string name="nc_location_current_position_description">Your current location</string>
 
+    <!-- voice messages -->
+    <string name="nc_voice_message_filename">Talk recording from</string>
```

Comment:
I would make the date and name placeholder part of the string, in case it is needed for a translation.
See https://github.com/nextcloud/spreed/pull/5610/files#diff-59506255a83039f3102821486034814a784ad7022780a1ebdb22146b184d3970R282 for inspiration